bin/crystal

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
As per [this stackoverflow answer](https://unix.stackexchange.com/a/77586/169750) this is fine:

> There are two programs whose location you can rely on on almost every unix variant: /bin/sh and /usr/bin/env.